generator/transit_generator.cpp

```diff
@@ -206,15 +206,12 @@ DeserializerFromJson::DeserializerFromJson(json_struct_t* node,
 
 void DeserializerFromJson::operator()(m2::PointD & p, char const * name)
 {
-  json_t * pointItem = nullptr;
-  if (name == nullptr)
-    pointItem = m_node; // Array item case
-  else
-    pointItem = my::GetJSONObligatoryField(m_node, name);
-
-  CHECK(json_is_object(pointItem), ());
-  FromJSONObject(pointItem, "x", p.x);
-  FromJSONObject(pointItem, "y", p.y);
+  GetTwoParamDict(name, "x", "y", p.x, p.y);
+}
+
+void DeserializerFromJson::operator()(ShapeId & id, char const * name)
+{
+  GetTwoParamDict(name, "stop1_id", "stop2_id", id.m_stop1Id, id.m_stop2Id);
```

Comment:
As you don't use name, please, comment it: ```c++ /* name */```.

Reply:
Could you please explain what you mean? I do use the param |name| in method `GetTwoParamDict(name, ...);`

Reply:
Ah, see now, sorry.